.github/workflows/presubmit.yml

```diff
@@ -3,47 +3,58 @@ name: Presubmit
 permissions:
   contents: read
 
-on: [push, pull_request]
+# Controls when the action will run.
+on:
+  push:
+  workflow_dispatch:
+  pull_request:
 
+# These jobs are potentially parallelizeable
 jobs:
   build:
-    name: Build all specs
+    name: Build spec artifacts
     runs-on: ubuntu-latest
+    # Refer to the build container by its SHA instead of the name, to
+    # prevent caching problems when updating the image.
+    # container: khronosgroup/docker-images:asciidoctor-spec.20240702
+    container: khronosgroup/docker-images@sha256:91a1b62399f9804e17110e89d85d62e90bc8d2ad7b94a329bee87784158368a9
```

Comment:
I'm not a docker expert, but this SHA appears to be from [asciidoctor-spec.20240627](https://hub.docker.com/layers/khronosgroup/docker-images/asciidoctor-spec.20240627/images/sha256-91a1b62399f9804e17110e89d85d62e90bc8d2ad7b94a329bee87784158368a9?context=explore) and not [asciidoctor-spec.20240702](https://hub.docker.com/layers/khronosgroup/docker-images/asciidoctor-spec.20240702/images/sha256-4aab96a03ef292439c9bd0f972adfa29cdf838d0909b1cb4ec2a6d7b2d14a37f?context=explore).  Should the comment be updated or should the SHA be updated, or am I completely misunderstanding how docker hub works?